internal/cmd/grants.go

```diff
@@ -15,12 +15,13 @@ import (
 )
 
 type grantsCmdOptions struct {
-	UserName  string
-	GroupName string
-	Resource  string
-	Role      string
-	Force     bool
-	Inherited bool
+	UserName    string
+	GroupName   string
+	Resource    string
```

Comment:
Possibly, we don't have a use case for it right now, but if we start to add different kinds of destinations it may be useful in the future. I think we should wait to see how we address the database table to remove the need for the `OR` in the query. That might give us a better idea of what to do about the `Resource` query parameter.